components/gpio_control/gpio_control.py

```diff
@@ -125,5 +125,6 @@ def gpio_loop(self):
     gpio_controler = gpio_control(phoniebox_function_calls)
 
     devices = gpio_controler.get_all_devices(config)
+    gpio_controler.checkDevicesDependencies()
```

Comment:
I could add a function that informs the Shutdown button about a configured StatusLED here. It could do other devices dependencies, too.
Informing means set the led status after a shutdown is aborted.
Hum then I don't need to check all this at the shutdown button and just use the value that has been given to the shutdown button... lets see how this looks like... at the next commit

Reply:
Done
https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1997/files#diff-3030f12dc8c3fa2e2c9c64dd6ca0ee1ae9a539a301e58e0211379e89ee17995cR141